pkg/apis/agones/v1/common.go

```diff
@@ -26,14 +26,18 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation/field"
 )
 
-// Block of const Error messages
+// nolint:stylecheck
```

Comment:
The linter is complaining that there's not a comment on exported variables. Is there a comment here that would make sense?

Reply:
Did it complaint on the two variables you added? but you already had comments for them, although there was a typo, maybe that's the cause?

Reply:
There was a comment that said "Error messages" that I removed because it was no longer a block of exclusively error messages since I added the two new const variables.

Reply:
Oh I see. Maybe something like "Block of const Error messages and GameServerAllocation Counter actions" if you don't want to add comments for each of the consts